istioctl/cmd/workload_test.go

```diff
@@ -31,6 +31,10 @@ import (
 	testKube "istio.io/istio/pkg/test/kube"
 )
 
+var (
+	fakeCACert = []byte("fake-CA-cert")
+)
+
```

Comment:
since we removed webhook_test.go, which declared the var: fakeCACert, which this workload_test.go is relying on, I'm creating one in this file to pass make it available.